extensions/wikia/AdEngine/js/video/player/porvata/porvata.js

```diff
@@ -80,6 +82,12 @@ define('ext.wikia.adEngine.video.player.porvata', [
 					if (!viewportListener) {
 						viewportListener = viewportObserver.addListener(params.container, inViewportCallback);
 					}
+					if (floater.canFloat(params)) {
+						isFloating = Boolean(floater.makeFloat(video, params, function() {
```

Comment:
`Boolean` on any object, even empty returns true, and `makeFloat` always returns object. Why do you want to do it like that?